cmd/thor/flags.go

```diff
@@ -128,4 +128,14 @@ var (
 		Name:  "skip-txpool-blocklist",
 		Usage: "skip applying txpool blocklist",
 	}
+	txPoolLengthFlag = cli.IntFlag{
+		Name:  "txpool-length",
+		Value: 10000,
+		Usage: "set txpool length",
+	}
+	txPoolLimitAccountFlag = cli.IntFlag{
+		Name:  "txpool-limit-account",
```

Comment:
suggested to be "txpool-limit-per-account"

Reply:
fixed.